spack/package.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Arbor(CMakePackage, CudaPackage):
+    """Arbor is a high-performance library for computational neuroscience
+    simulations."""
+
+    homepage = 'https://github.com/arbor-sim/arbor/'
```

Comment:
Shall we start using arbor-sim.github.io?

Reply:
We can.
Though, I have to make some changes to that site (e.g. stop it from opening up links in new tabs).